src/megaapi_impl.cpp

```diff
@@ -23073,6 +23081,45 @@ void MegaApiImpl::sendPendingRequests()
                                                            request->getNodeHandle()));
             break;
         }
+        case MegaRequest::TYPE_FETCH_GOOGLE_ADS:
+        {
+                MegaStringList* stringList = request->getMegaStringList();
+                std::vector<std::string> vectorString;
```

Comment:
Maybe, instead of adding a copy, we could add a getter at the private class, and get the `MegaRequestPrivate::mList`  directly thanks to a casting+getter, avoiding the copy.